src/plugins/data/common/index_patterns/index_patterns/ensure_default_index_pattern.ts

```diff
@@ -30,14 +30,13 @@
 
 import { includes } from 'lodash';
 import { IndexPatternsContract } from './index_patterns';
-import { UiSettingsCommon } from '../types';
-
```

Comment:
Nit: we'd better leave the empty line between the import section and code section.